content/it/docs/reference/glossary/kube-scheduler.md

```diff
@@ -0,0 +1,18 @@
+---
+title: kube-scheduler
+id: kube-scheduler
+date: 2018-04-12
+full_link: /docs/reference/generated/kube-scheduler/
+short_description: >
+  Componente che gira sui master node che controlla i pod appena creati che non hanno un nodo assegnato, e dopo averlo identificato glielo assegna.
+
+aka: 
+tags:
+- architecture
+---
+ Componente che gira sui master node che controlla i pod appena creati che non hanno un nodo assegnato, e dopo averlo identificato glielo assegna.
+
+<!--more--> 
+
+I fattori presi in considerazioni nella programmazione di un pod includono le risorse richieste individualmente e collettivamente, i vincoli delle hardware/software/policy, le indicazioni di affinità e di anti-affinità, il collocamento dei dati, le interferenze tra diversi workload e le scadenze.
```

Comment:
nella programmazione di un pod --> nell'individuare un nodo a cui assegnare un Pod
le risorse richieste individualmente e collettivamente --> la richiesta di risorse del Pod/degli altri workload presenti nel sistema
affinità e di anti-affinità --> affinity and anti-affinity (non tradurrei temini che si mappano su API objects)
il collocamento dei dati --> requisiti relativi alla disponibilità di dati/Volumes